meson.build

```diff
@@ -2676,6 +2678,10 @@ public_programs += exe
 
 executable('systemd-shutdown',
            systemd_shutdown_sources,
+           'src/core/mount-setup.c',
+           'src/core/mount-setup.h',
+           'src/core/killall.c',
+           'src/core/killall.h',
```

Comment:
This is not nice. We should make a convenience library to compile every file exactly once. (We make an exception for tests, but for "real" binaries, this is true). But we can fix this up later.